fern/versions/v2025-02-24/pages/disbursements/disbursement-lifecycle.mdx

```diff
@@ -33,23 +33,43 @@ Once a Disbursement contains a `submission`, a payment has been initiated throug
 
 <AccordionGroup>
     <Accordion title="Account Transfer">
-        A Account Transfer happens instantly and can be sent 24 hours a day, 7 days a week. If a nonprofit has an account with Chariot, Chariot will send funds using this payment rail.
+        A Account Transfer happens instantly and can be sent 24 hours a day, 7 days a week.
+        If a nonprofit has an account with Chariot, Chariot will typically send funds using this payment rail.
 
         Since Account Transfers are instant, they don't have a payment lifecycle and funds are available to the nonprofit immediately.
     </Accordion>
+    <Accordion title="ACH">
+        ACH submission represents an credit (push) payment using the Automated Clearing House (ACH) network.
+        This submission method is used if the receiving organization is not avaiable for Account Transfers.
+        Funds availability is typically 1-2 days.
+
+        A common frustration with ACH is that there’s no transfer receipt notification method.
+        As a transfer originator, you have no way to definitely know if or when a transfer has arrived to a recipient’s account.
+        <Frame>
+            <img src='../../images/ach_lifecycle.png' />
+        </Frame>
+        | Status | Description |
+        |--------|-------------|
+        | `initiated` | The ACH transfer has been initiated and is pending submission to the Federal Reserve |
+        | `rejected` | The ACH transfer was rejected |
+        | `submitted` | The ACH transfer has been submitted to the Federal Reserve |
+        | `returned` | The ACH transfer was returned by the receiving organization |
+    </Accordion>
     <Accordion title="Mailed Check">
-    A check submission represents a paper check that is sent to the receiving organization by mail. This is the submission method used when the receiving organization does not have a Chariot account.
-<Frame>
-    <img src='../../images/check_lifecycle.png' />
-</Frame>
+        A check submission represents a paper check that is sent to the receiving organization by mail.
```

Comment:
can we say: "A check submission represents a physical check that is mailed to the receiving organization."